datafusion/functions/src/crypto/md5.rs

```diff
@@ -64,11 +64,11 @@ impl ScalarUDFImpl for Md5Func {
     fn return_type(&self, arg_types: &[DataType]) -> Result<DataType> {
         use DataType::*;
         Ok(match &arg_types[0] {
-            LargeUtf8 | LargeBinary => LargeUtf8,
+            LargeUtf8 | LargeBinary => Utf8,
```

Comment:
👍

Reply:
Maybe as a follow on it should really be LargeUtf8 but then we would have to change the function I suppose